admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -10,8 +10,8 @@ is usually not necessary to edit ``config/config.php``.
 
 .. note:: The installer creates a configuration containing the essential parameters.
    Only manually add configuration parameters to ``config/config.php`` if you need to
-   use a special value for a parameter. **Do not copy everything from
-   ``config/config.sample.php``. Only enter the parameters you wish to modify!**
+   use a special value for a parameter. **Do not copy everything from**
+   ``config/config.sample.php``**. Only enter the parameters you wish to modify!**
 
```

Comment:
Wow, that's a fun one! On my local build the only way it looks right is to have a space between the double asterisks and the double backticks, like this:

    **Do not copy everything from** ``config/config.sample.php`` **. Only enter the parameters you wish to modify!**

Then config/config.sample.php is formatted as code, and the surrounding text is bold. I think you need a space between the closing backticks and asterisks.